Dockerfile

```diff
@@ -7,7 +7,6 @@ COPY cdn-uploader.js .
 RUN npm init --yes
 RUN npm install basic-ftp
 RUN mkdir /source
-VOLUME /source
```

Comment:
Why did you remove it? I would like to declare the volume here in the same way that we are declaring the environment variables to document what we need to configure to _run_ this image.